docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,142 @@
+
+
+Rationale
+---------
+
+Documentation projects may be kept online indefinitely, even though a newer version of the same project exists elsewhere.
+There are many reasons this can happen,
+including forks, old official docs that are unmaintained, and many other situations.
+
+The problem with old, outdated docs is that users will find them in search results,
+and get confused to the validity of them.
+Projects will then get support requests from people who are using an old and incorrect documentation version.
+
+We have this policy to allow a reporter to request the *delisting* of forks that are old and outdated.
+
+
+High level overview
+~~~~~~~~~~~~~~~~~~~
+
+The process at a high level looks like:
+
+* A reporter emails us about a project they think is outdated and unofficial
+* An RTD team member evaluates it to make sure it's outdated and unofficial, according to this policy
+* We delist this project from search results and send an email to owners of the RTD project
+* If a project owner objects, we evaluate their evidence and make a final decision
+
+
+Definitions
+-----------
+
+
+Source project
+~~~~~~~~~~~~~~
+
+For a given documentation project to be assessed, the *source project* comes into question. This is defined as:
+
+* The origin project from which the documentation project's repository was forked (if forked).
+* The origin project from which the documentation project was copied (if not forked but evidently copied).
+* The original project which the documentation project refers as the subject of documentation.
+* Any other project that is understood by the core team to be a the *source project*.
+
+
+Unofficial projects
+~~~~~~~~~~~~~~~~~~~
+
+A project is considered *unofficial* when it is not linked to or mentioned in *any* of the following places:
+
+* Websites and domains associated with the project
+* The project's primary repository -- README files, repository description, or source code
+
+
+Unmaintained projects
+~~~~~~~~~~~~~~~~~~~~~
+
+A project is considered *unmaintained* when ANY of the following are met:
+
+* The configured version control repository is unreadable. This can happen if the repository is deleted, credentials are broken or the Git host is permanently unresponsive.
+* The project is only serving content from releases and commits 6 months or older than its source repository.
+* All builds have failed for more than 6 months.
+
+
+Reachability
+~~~~~~~~~~~~
+
+In every case where contacting the user is necessary, the core team will reach out using one of the following means of contact:
+
+* E-mail address on file in the user's profile
+* E-mail address on the GitHub user account where the project is pointed
+
+The core team will stop trying to reach the user after 3 weeks and the user will be considered *unreachable*.
+
+
+Implementation
+--------------
+
+
+Delisting of an abandoned project
```

Comment:
This title shouldn't use the words "abandoned project" since that's a different policy. If any, it should say "unofficial and unmaintained"